source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -278,8 +293,44 @@ HTTP Webhook Audit Log Target
 
       Requires specifying :mc-conf:`~audit_webhook.client_cert`.
 
-      This setting corresponds to the
-      :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+      This setting corresponds to the :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+
+   .. mc-conf:: queue_dir
+
+      .. versionadded:: RELEASE.2023-05-18T00-05-36Z
+
+      *Optional*
+
+      Specify the directory path to enable MinIO's persistent event store for undelivered messages, such as ``/home/events``.
+
+      MinIO stores undelivered events in the specified store while the webhook service is offline and replays the stored events when connectivity resumes.
+
+      This setting corresponds to the :envvar:`MINIO_AUDIT_WEBHOOK_QUEUE_DIR` environment variable.
+
+   .. mc-conf:: kafka_queue_dir
+
+      .. versionadded:: RELEASE.2023-05-18T00-05-36Z
+
+      *Optional*
+
+      Specify the directory path to enable MinIO's persistent event store for undelivered messages, such as ``/home/events``.
+
+      MinIO stores undelivered events in the specified store while the Kafka service is offline and replays the stored events when connectivity resumes.
+
+      This setting corresponds to the :envvar:`MINIO_AUDIT_KAFKA_QUEUE_DIR` environment variable.
+
+   .. mc-conf:: kafka_queue_size
```

Comment:
I made up this name because I don't know what the corresponding command is for `MINIO_AUDIT_KAFKA_QUEUE_SIZE`

Answer TBD

Reply:
On hold, see https://github.com/minio/docs/issues/886